### ISSUE TITLE
Ignore redirects for slash in key/value and "value in key page" and "no value for tag page" Wiki Problems

### ISSUE THREAD

**Issue by JesseWeinstein** [taginfo/taginfo#416]
On the https://wiki.openstreetmap.org/w/index.php?title=Taginfo/Parsing_the_Wiki page, there are 4 issues listed:
"slash in key", "slash in value", "value in key page" and "no value for tag page" that, because they don't ignore redirects, cannot be resolved except by wiki admins who can delete pages. It seems like it would be much more useful to have a list of non-redirects with these problems, and ignore redirect pages.

**Comment by joto** [comment]
I don't agree. Bad wiki pages should be deleted, not worked around by redirects. Redirects are a huge problem for taginfo, because you can not know whether a redirect contains some useful information (this tag is deprecated and was renamed to this other tag) or whether it was just a typo somebody corrected and no admin could be bothered to remove the original page.

**Comment by JesseWeinstein** [comment]
OK, fair enough. But it would still be useful to have a distinction between redirected and non-redirected pages, so it was easier to tell which ones needed admin assistance. I think, ideally, no tag pages should be hard-redirects -- they all should contain Description templates, and just have soft redirects in the non-templated descriptions to pages that describe the whole of the tagging scheme of which they are a part. But that's a separate issue.
